garak/resources/theme/__init__.py

```diff
@@ -4,3 +4,8 @@
 PROBE_RGB = "e5a70e"
 DETECTOR_RGB = "2c79da"
 GENERATOR_RGB = "c061cb"
+
+EMOJI_SCALE_FACE = "😭🙁😐🙂🤩"
```

Comment:
I LIKE IT A LOT

Reply:
People who liked this content also liked: https://twitter.com/peterjansen_ai/status/1818894729633055211